src/sentry/static/sentry/app/components/avatar/baseAvatar.jsx

```diff
@@ -135,10 +136,14 @@ class BaseAvatar extends React.Component {
         >
           {this.state.showBackupAvatar && this.renderLetterAvatar()}
           {this.renderImg()}
-        </span>
+        </StyledBaseAvatar>
       </Tooltip>
     );
   }
 }
 
 export default BaseAvatar;
+
+const StyledBaseAvatar = styled('span')`
+  flex-shrink: 0;
```

Comment:
This is less than ideal, since `flex-shrink` is hinting at a flex layout that isn't present in this template. Ideally, we should keep flex layouts as self-contained as possible but if this fixes a bug then maybe whatever for now

Reply:
Agreed, but there won't be any side effects, and I *think* this is a pretty good default in the case of flex layouts? Would clarifying this in a comment help ease some concerns?